File: sprixel/misc/value__S_quote.js

// 
Act.types.value__S_quote = function(){
    //throw ;
    //throw keys(this.node);
    do { switch(this.phase) {
    case 0:
        this.phase = 1;
        return this.next = new Act(this.node.quote, this);
    case 1:
        this.result = this.next.result;
    default: break;
    } } while (false);
    this.next = null;
    return this.invoker;
};
1;

File: vill/src/value__S_quote.cpp
// value__S_quote.cpp
#include "villCompiler.h"

Value *
value__S_quote( struct villCompiler * vill, struct graph_node * node ) {
  Value * result = NULL;
  vill -> debug_flags & DEBUG_USER && fprintf( stderr, "<value__S_quote>" );
  graph_node * quote = ast_mapentry( node, "quote" );
  function_pointer Codegen;
  Codegen = (function_pointer) quote -> data;
  assert( Codegen != NULL );
  result = Codegen( vill, quote );
  vill -> debug_flags & DEBUG_USER && fprintf( stderr, "</value__S_quote>" );
  return result;
}
